src/main/java/io/appium/java_client/ios/IOSDeviceActionShortcuts.java

```diff
@@ -56,5 +57,14 @@ default void hideKeyboard(String strategy, String keyName) {
     default void shake() {
         CommandExecutionHelper.execute(this, shakeCommand());
     }
+    
```

Comment:
I am against the adding to the `IOSDeviceActionShortcuts`. This interface is going to be deprecated. Please take a look [at this working branch](https://github.com/TikhomirovSergey/java-client/tree/SrinivasanTarget-TouchActionSplitting/src/main/java/io/appium/java_client/ios). [IOSDeviceActionShortcuts](https://github.com/TikhomirovSergey/java-client/blob/SrinivasanTarget-TouchActionSplitting/src/main/java/io/appium/java_client/ios/IOSDeviceActionShortcuts.java) is going to be deprecated. 

It is better to add such interface:

```java
package io.appium.java_client.ios;

public interface ChecksTouchId extends ExecutesMethod {

    /**
    * Simulate touchId event
    *
    * @param match If true, simulates a successful fingerprint scan. If false, simulates a failed fingerprint scan
    */     
    default void touchId(boolean match) {
        CommandExecutionHelper.execute(this, touchIdCommand(match));
    }
}
```

and make IOSDriver implement it.